graylog2-web-interface/src/pages/NodesPage.jsx

```diff
@@ -28,7 +28,7 @@ const NodesPage = createReactClass({
     }
     if (this._hasExternalURI()) {
       return (
-        <ExternalLinkButton bsStyle="info" href={URLUtils.qualifyUrl(Routes.GLOBAL_API_BROWSER)}>
+        <ExternalLinkButton bsStyle="info" href={URLUtils.qualifyUrl('/api-browser/global/index.html')}>
```

Comment:
Can you please extract it into a constant in the same file? That makes it way more visible.